c_api/cpp_demangle.h

```diff
@@ -0,0 +1,16 @@
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// Flags currently in `cpp_demangle`
+typedef enum {
+  DMGL_NO_OPTS = 0,
+  DMGL_PARAMS  = (1 << 0)
+} DemangleOptions;
```

Comment:
It probably makes sense to use a struct since there will eventually be non-bool options that won't fit here, like maximum recursion depth and maximum demangled symbol length (to avoid exponential symbols DoS).

Reply:
That makes sense, fixed.